package.json

```
@@ -12,6 +12,7 @@
   },
   "scripts": {
     "format": "prettier '**/{*.{js?(on),ts?(x),graphql,md,scss},.*.js?(on)}' --write --list-different --config prettier.config.js",
+    "format:changed": "prettier $(git diff --diff-filter=d --name-only origin/main... & git ls-files --other --modified --exclude-standard | grep -E '\\.(js|json|ts|tsx|graphql|md|scss)$' | xargs) --write --list-different --config prettier.config.js",
```

I think this should be `&&` so that it waits for the output (and not spawn it detached?)

```suggestion
    "format:changed": "prettier $(git diff --diff-filter=d --name-only origin/main... && git ls-files --other --modified --exclude-standard | grep -E '\\.(js|json|ts|tsx|graphql|md|scss)$' | xargs) --write --list-different --config prettier.config.js",
```

I tested the command locally with `--no-pager` and it seems to work (you need to remove the double escape: `\\.` => `\.` to repro)

---

Oh, thanks for the && 
Are you saying we should run git diff with the `--no-pager` param?

---

> Are you saying we should run git diff with the --no-pager param?

Oh, no, in the script this is not necessary. I was just leaving this as context of how I tested it 🙈